### PR TITLE
Object inspection support

### PR DESCRIPTION
**Pull request by twelho**
This PR adds the `ignite inspect` command to retrieve the information/configuration of an Ignite object.

### CODE REVIEW

File: cmd/ignite/cmd/inspect.go [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+			}())
+		},
+	}
+
+	addInspectFlags(cmd.Flags(), i)
+	return cmd
+}
+
+func addInspectFlags(fs *pflag.FlagSet, i *run.InspectFlags) {
+	fs.BoolVarP(&i.YAMLOutput, "yaml", "y", false, "Output the object in YAML format instead of JSON")
```
Comment (luxas) [inline cmd/ignite/cmd/inspect.go]: doesn't have to be done now, but eventually this should be done as `--output=<type>` => `-o=yaml`, `-o=json` instead for reusability and consistency.

File: cmd/ignite/cmd/inspect.go [hunk 2]
```diff
@@ -0,0 +1,44 @@
+package cmd
+
+import (
+	"github.com/lithammer/dedent"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	"github.com/weaveworks/ignite/cmd/ignite/run"
+	"github.com/weaveworks/ignite/pkg/errutils"
+	"io"
```
Comment (luxas) [inline cmd/ignite/cmd/inspect.go]: should be in its own group up top

File: cmd/ignite/run/inspect.go [hunk 3]
```diff
@@ -0,0 +1,61 @@
+package run
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/weaveworks/ignite/pkg/apis/ignite/scheme"
+	"github.com/weaveworks/ignite/pkg/client"
+	"github.com/weaveworks/ignite/pkg/filter"
```
Comment (luxas) [inline cmd/ignite/run/inspect.go]: nit: grouping

File: cmd/ignite/run/inspect.go [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+	*InspectFlags
+	object meta.Object
+}
+
+func (i *InspectFlags) NewInspectOptions(k, objectMatch string) (*inspectOptions, error) {
+	var err error
+	var kind meta.Kind
+	io := &inspectOptions{InspectFlags: i}
+
+	switch k {
```
Comment (luxas) [inline cmd/ignite/run/inspect.go]: `strings.ToLower(k)` to support `ignite inspect Image foo` if somebody would do it. no need to error in that case

File: cmd/ignite/cmd/inspect.go (hunk 1 above)
Comment (twelho) [reply cmd/ignite/cmd/inspect.go]: Reworked in c6622cf.

File: cmd/ignite/cmd/inspect.go (hunk 2 above)
Comment (twelho) [reply cmd/ignite/cmd/inspect.go]: Fixed in c6622cf.

File: cmd/ignite/run/inspect.go (hunk 3 above)
Comment (twelho) [reply cmd/ignite/run/inspect.go]: Fixed in c6622cf.

File: cmd/ignite/run/inspect.go (hunk 4 above)
Comment (twelho) [reply cmd/ignite/run/inspect.go]: Yes, was not sure if it's needed as it could train the user "incorrect" usage. Implemented in c6622cf now.

**Pull request merged**